client/src/style.css

```diff
@@ -4,8 +4,6 @@
   height: 338px;
   flex-direction: row;
```

Comment:
I believe flex-direction at "row" is the default behavior, so this is unnecessary?